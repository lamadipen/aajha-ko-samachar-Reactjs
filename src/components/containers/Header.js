import React from 'react';
import { Link } from "react-router-dom";

const Header = () => {
    return (
        <header className="header">

            <nav className="navbar navbar-static-top" role="navigation">
                <div className="navbar-left">
                    <ul className="nav navbar-nav">
                        <li className="notifications-menu">
                            <Link to="/home" style={{color: 'white', background: '#303030'} } className="logo" >
                                Aajha ko Samachar
                            </Link>
                        </li>
                        <li className="notifications-menu">
                            <Link to="/home">
                                <span>Home</span>
                            </Link>
                        </li>
                        <li className="notifications-menu">
                            <Link to="/aboutUs">
                                <span>About Us</span>
                            </Link>
                        </li>
                        <li className="notifications-menu">
                            <Link to="/post">
                                <span>Post</span>
                            </Link>
                        </li>
                        <li className="notifications-menu">
                            <Link to="/blog">
                                <span>Blog</span>
                            </Link>
                        </li>
                    </ul>
                </div>
            </nav>
        </header>
    )
}

export default Header;