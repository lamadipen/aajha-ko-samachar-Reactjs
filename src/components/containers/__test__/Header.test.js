import React from 'react';
import { shallow } from  'enzyme';
import Header from '../Header';


describe('Header',() => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Header/>);
    })

    it('run without crashing', () => {
         expect(wrapper).toMatchSnapshot();
    })

})

