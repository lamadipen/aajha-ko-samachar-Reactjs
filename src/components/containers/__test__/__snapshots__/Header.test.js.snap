// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header run without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav
        className="navbar navbar-static-top"
        role="navigation"
      >
        <div
          className="navbar-left"
        >
          <ul
            className="nav navbar-nav"
          >
            <li
              className="notifications-menu"
            >
              <Link
                className="logo"
                replace={false}
                style={
                  Object {
                    "background": "#303030",
                    "color": "white",
                  }
                }
                to="/home"
              >
                Aajha ko Samachar
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/home"
              >
                <span>
                  Home
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/aboutUs"
              >
                <span>
                  About Us
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/post"
              >
                <span>
                  Post
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/blog"
              >
                <span>
                  Blog
                </span>
              </Link>
            </li>
          </ul>
        </div>
      </nav>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="navbar-left"
        >
          <ul
            className="nav navbar-nav"
          >
            <li
              className="notifications-menu"
            >
              <Link
                className="logo"
                replace={false}
                style={
                  Object {
                    "background": "#303030",
                    "color": "white",
                  }
                }
                to="/home"
              >
                Aajha ko Samachar
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/home"
              >
                <span>
                  Home
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/aboutUs"
              >
                <span>
                  About Us
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/post"
              >
                <span>
                  Post
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/blog"
              >
                <span>
                  Blog
                </span>
              </Link>
            </li>
          </ul>
        </div>,
        "className": "navbar navbar-static-top",
        "role": "navigation",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <ul
            className="nav navbar-nav"
          >
            <li
              className="notifications-menu"
            >
              <Link
                className="logo"
                replace={false}
                style={
                  Object {
                    "background": "#303030",
                    "color": "white",
                  }
                }
                to="/home"
              >
                Aajha ko Samachar
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/home"
              >
                <span>
                  Home
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/aboutUs"
              >
                <span>
                  About Us
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/post"
              >
                <span>
                  Post
                </span>
              </Link>
            </li>
            <li
              className="notifications-menu"
            >
              <Link
                replace={false}
                to="/blog"
              >
                <span>
                  Blog
                </span>
              </Link>
            </li>
          </ul>,
          "className": "navbar-left",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="notifications-menu"
              >
                <Link
                  className="logo"
                  replace={false}
                  style={
                    Object {
                      "background": "#303030",
                      "color": "white",
                    }
                  }
                  to="/home"
                >
                  Aajha ko Samachar
                </Link>
              </li>,
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/home"
                >
                  <span>
                    Home
                  </span>
                </Link>
              </li>,
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/aboutUs"
                >
                  <span>
                    About Us
                  </span>
                </Link>
              </li>,
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/post"
                >
                  <span>
                    Post
                  </span>
                </Link>
              </li>,
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/blog"
                >
                  <span>
                    Blog
                  </span>
                </Link>
              </li>,
            ],
            "className": "nav navbar-nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  className="logo"
                  replace={false}
                  style={
                    Object {
                      "background": "#303030",
                      "color": "white",
                    }
                  }
                  to="/home"
                >
                  Aajha ko Samachar
                </Link>,
                "className": "notifications-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Aajha ko Samachar",
                  "className": "logo",
                  "replace": false,
                  "style": Object {
                    "background": "#303030",
                    "color": "white",
                  },
                  "to": "/home",
                },
                "ref": null,
                "rendered": "Aajha ko Samachar",
                "type": [Function],
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/home"
                >
                  <span>
                    Home
                  </span>
                </Link>,
                "className": "notifications-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <span>
                    Home
                  </span>,
                  "replace": false,
                  "to": "/home",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Home",
                  },
                  "ref": null,
                  "rendered": "Home",
                  "type": "span",
                },
                "type": [Function],
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/aboutUs"
                >
                  <span>
                    About Us
                  </span>
                </Link>,
                "className": "notifications-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <span>
                    About Us
                  </span>,
                  "replace": false,
                  "to": "/aboutUs",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "About Us",
                  },
                  "ref": null,
                  "rendered": "About Us",
                  "type": "span",
                },
                "type": [Function],
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/post"
                >
                  <span>
                    Post
                  </span>
                </Link>,
                "className": "notifications-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <span>
                    Post
                  </span>,
                  "replace": false,
                  "to": "/post",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Post",
                  },
                  "ref": null,
                  "rendered": "Post",
                  "type": "span",
                },
                "type": [Function],
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/blog"
                >
                  <span>
                    Blog
                  </span>
                </Link>,
                "className": "notifications-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <span>
                    Blog
                  </span>,
                  "replace": false,
                  "to": "/blog",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Blog",
                  },
                  "ref": null,
                  "rendered": "Blog",
                  "type": "span",
                },
                "type": [Function],
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
        "type": "div",
      },
      "type": "nav",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav
          className="navbar navbar-static-top"
          role="navigation"
        >
          <div
            className="navbar-left"
          >
            <ul
              className="nav navbar-nav"
            >
              <li
                className="notifications-menu"
              >
                <Link
                  className="logo"
                  replace={false}
                  style={
                    Object {
                      "background": "#303030",
                      "color": "white",
                    }
                  }
                  to="/home"
                >
                  Aajha ko Samachar
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/home"
                >
                  <span>
                    Home
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/aboutUs"
                >
                  <span>
                    About Us
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/post"
                >
                  <span>
                    Post
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/blog"
                >
                  <span>
                    Blog
                  </span>
                </Link>
              </li>
            </ul>
          </div>
        </nav>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="navbar-left"
          >
            <ul
              className="nav navbar-nav"
            >
              <li
                className="notifications-menu"
              >
                <Link
                  className="logo"
                  replace={false}
                  style={
                    Object {
                      "background": "#303030",
                      "color": "white",
                    }
                  }
                  to="/home"
                >
                  Aajha ko Samachar
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/home"
                >
                  <span>
                    Home
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/aboutUs"
                >
                  <span>
                    About Us
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/post"
                >
                  <span>
                    Post
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/blog"
                >
                  <span>
                    Blog
                  </span>
                </Link>
              </li>
            </ul>
          </div>,
          "className": "navbar navbar-static-top",
          "role": "navigation",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul
              className="nav navbar-nav"
            >
              <li
                className="notifications-menu"
              >
                <Link
                  className="logo"
                  replace={false}
                  style={
                    Object {
                      "background": "#303030",
                      "color": "white",
                    }
                  }
                  to="/home"
                >
                  Aajha ko Samachar
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/home"
                >
                  <span>
                    Home
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/aboutUs"
                >
                  <span>
                    About Us
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/post"
                >
                  <span>
                    Post
                  </span>
                </Link>
              </li>
              <li
                className="notifications-menu"
              >
                <Link
                  replace={false}
                  to="/blog"
                >
                  <span>
                    Blog
                  </span>
                </Link>
              </li>
            </ul>,
            "className": "navbar-left",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="notifications-menu"
                >
                  <Link
                    className="logo"
                    replace={false}
                    style={
                      Object {
                        "background": "#303030",
                        "color": "white",
                      }
                    }
                    to="/home"
                  >
                    Aajha ko Samachar
                  </Link>
                </li>,
                <li
                  className="notifications-menu"
                >
                  <Link
                    replace={false}
                    to="/home"
                  >
                    <span>
                      Home
                    </span>
                  </Link>
                </li>,
                <li
                  className="notifications-menu"
                >
                  <Link
                    replace={false}
                    to="/aboutUs"
                  >
                    <span>
                      About Us
                    </span>
                  </Link>
                </li>,
                <li
                  className="notifications-menu"
                >
                  <Link
                    replace={false}
                    to="/post"
                  >
                    <span>
                      Post
                    </span>
                  </Link>
                </li>,
                <li
                  className="notifications-menu"
                >
                  <Link
                    replace={false}
                    to="/blog"
                  >
                    <span>
                      Blog
                    </span>
                  </Link>
                </li>,
              ],
              "className": "nav navbar-nav",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="logo"
                    replace={false}
                    style={
                      Object {
                        "background": "#303030",
                        "color": "white",
                      }
                    }
                    to="/home"
                  >
                    Aajha ko Samachar
                  </Link>,
                  "className": "notifications-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Aajha ko Samachar",
                    "className": "logo",
                    "replace": false,
                    "style": Object {
                      "background": "#303030",
                      "color": "white",
                    },
                    "to": "/home",
                  },
                  "ref": null,
                  "rendered": "Aajha ko Samachar",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/home"
                  >
                    <span>
                      Home
                    </span>
                  </Link>,
                  "className": "notifications-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <span>
                      Home
                    </span>,
                    "replace": false,
                    "to": "/home",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Home",
                    },
                    "ref": null,
                    "rendered": "Home",
                    "type": "span",
                  },
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/aboutUs"
                  >
                    <span>
                      About Us
                    </span>
                  </Link>,
                  "className": "notifications-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <span>
                      About Us
                    </span>,
                    "replace": false,
                    "to": "/aboutUs",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "About Us",
                    },
                    "ref": null,
                    "rendered": "About Us",
                    "type": "span",
                  },
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/post"
                  >
                    <span>
                      Post
                    </span>
                  </Link>,
                  "className": "notifications-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <span>
                      Post
                    </span>,
                    "replace": false,
                    "to": "/post",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Post",
                    },
                    "ref": null,
                    "rendered": "Post",
                    "type": "span",
                  },
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/blog"
                  >
                    <span>
                      Blog
                    </span>
                  </Link>,
                  "className": "notifications-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <span>
                      Blog
                    </span>,
                    "replace": false,
                    "to": "/blog",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Blog",
                    },
                    "ref": null,
                    "rendered": "Blog",
                    "type": "span",
                  },
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          "type": "div",
        },
        "type": "nav",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
